### DISCUSSION TITLE
make_trade for v3 sends to 0x0 recipient

### DISCUSSION THREAD

**Discussion by ToposFinance** [uniswap-python/uniswap-python#120] [General]
hello, when you drill down to the final function that is called when ETH is first token, it seems you have hardcoated in a recipient address.

**Comment by ErikBjare** [comment]
Holy shit, no idea how I could let this slip! Thanks for letting me know!

Fixed in https://github.com/uniswap-python/uniswap-python/commit/371eb849dcc0ad4387f3628b9e5278939641cc03

Also published a new release (0.5.2) to PyPI which contains the fix.

**Reply by ToposFinance** [reply]
thanks. it was an expensive lesson for me...;)
